pipelinesScripts/validator/utils/utils_test.go

```diff
@@ -0,0 +1,19 @@
+package utils
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestExtractRepoDetails(t *testing.T) {
+	owner, repo := ExtractRepoDetails("https://github.com/JFrog/Jfrog-ClI-plugins")
+	assert.Equal(t, "jfrog", owner)
+	assert.Equal(t, "jfrog-cli-plugins", repo)
+}
+
+func TestGetPluginsDescriptor(t *testing.T) {
```

Comment:
Shouldn't this test function start with a clone command? Doesn't it find zero descriptor the way it is right now?

Reply:
No, remember that validator is located inside `jfrog-cli-plugins-reg` project, so no need to clone this project.